packages/graphql/src/public/client.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
with this mapping, the client knows which hash to send to the server for which of the graphql ops